library/core/src/lib.rs

```diff
@@ -140,6 +140,7 @@
 #![feature(const_default_impls)]
 #![feature(duration_consts_float)]
 #![feature(maybe_uninit_uninit_array)]
+#![cfg_attr(not(bootstrap), feature(const_str_ptr_len))]
```

Comment:
Could you move this a few lines up so it's sorted correctly between the other const_ features? Thanks!